src/modules/market/components/market-trading--confirm/market-trading--confirm.jsx

```diff
@@ -94,11 +94,12 @@ const MarketTradingConfirm = (p) => {
           onClick={(e) => {
             p.market.onSubmitPlaceTrade(p.selectedOutcome.id, (err, tradeGroupID) => {
               // onSent/onFailed CB
-              if (!err) p.toggleShowOrderPlaced()
+              if (!err) {
+                p.toggleShowOrderPlaced()
```

Comment:
Hm, is a toggle the desired behavior here?  If I submit two order tickets quickly, it doesn't seem correct to have the second order ticket submission hide the `showOrderPlaced` box.

Reply:
this should toggle extremely quickly as `onSent` usually comes back immediately, however I'll move it out to just passed the `p.prevPage` call so that it alway shows that "order placed!" ribbon once we have successfully called `placeTrade` regardless of if the `onSent` returned an error. They should still see notifications indicating a failure.

Reply:
oh i misunderstood, i see what you mean, it shouldn't toggle, it should force that it's shown. Gotcha. will adjust